app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name, age):
+        self.age = age
+        self.name = name
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    lst_of_perps = [Person(person["name"], person["age"]) for person in people]
+
+    for i in people:
```

Comment:
Could you use more informative names for loop variable?
For example, `person` or `person_dict`